lib/src/store/picture_storage.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:get/get.dart';
+import 'package:sqflite/sqflite.dart';
+import 'dart:io';
+
+class PictureStorage {
+  static pictureInformationMap(String courseId, String label, String picturePath, String note) {
+    return {
+      'courseId': courseId,
+      'label': label,
+      'note': note,
+      'picturePath': picturePath,
+    };
+  }
+
+  static takePictureToStorage(String courseId, String picturePath) async {
+    Database pictureDB = Get.find<Database>();
+    final information = pictureInformationMap(courseId, "unlabeled", picturePath, "");
+    await pictureDB.insert(
+      "photo_storage",
+      information,
+      conflictAlgorithm: ConflictAlgorithm.replace,
+    );
+  }
+
+  static getCoursePicture(String courseId) async {
+    Database pictureDB = Get.find<Database>();
+    List<Map> picturePaths =  await pictureDB.rawQuery('SELECT  * '
+                          'FROM photo_storage '
+                          'WHERE courseId=$courseId');
+    return picturePaths;
+  }
```

Comment:
There are several problems here:
1. You should give this method a return type.
2. It may be better to specify the key and value type for `Map` for picture paths, it may be `Map<String, dynamic>` in my opinion.
3. The `return` can be combined with the previous line.
4. Check your formatting before you commit, you accidentally added an extra space before `await`.

After reviewing these, this method can be improved in this way:
```suggestion
  static Map<String, dynamic> getCoursePicture(String courseId) async {
    final pictureDB = Get.find<Database>();
    return await pictureDB.rawQuery('SELECT  * '
                          'FROM photo_storage '
                          'WHERE courseId=$courseId');
  }
```